domains/hongson.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "4h3w",
+           "discord": "1221370693959286892",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.qIYzfjX8SF6w7P66KGOYXYcXPsh4FzZ-qiyI9JEA0A0Efdk1y3jO6VUaWdrTkknWfda7cbFTg4tUYxQtUPug-K_nRAP6H29OPLbOnZFosft6f3gl-m1ERIDANeF6PihgjXMnmdb86nMjetOib332kQkbdF0c0T6VYTcj107uazoAZ8o-I2A8wt63rjUn9yioZfqgevoihJV7T57UdyPFF1VqkHd_E4gW1Pa47p3wWXMVMySRg7WkVC4cZzD4eJSOoitwU7pi-COfCmUPojDVydpLHGqho2us618V3Lh2Jd9iAgqPGlKUYoknJMjfqUsm1SknqiH2H8LnZGL6XfM_1A.CPDDTmH6eHfeDFdSqafq5g.SFZ3vDgUzGwWTyJHhEdNUPenuebckQolGRX1nhMPi8hQKOeijQLdn5WwYfz-jje1v1gcmG7RVapuX4PaqRKj3g.KH-62lDIkbTJtEP9aJ6Ctg"
+        },
+    
+        "record": {
+            "CNAME": "ariella.ns.cloudflare.com"
```

Comment:
That's a NS record, they should be in a NS record field, which are under stricter rules.